facturacion_mexico/facturacion_fiscal/timbrado_api.py

```
@@ -2471,6 +2485,31 @@ def _validate_payload_completeness_ro(self, invoice_data, sales_invoice):
+				{"sales_invoice": factura_fiscal.get("sales_invoice")},
+				"name",
+			)
+			if ffm_name:
+				frappe.db.set_value(
+					"Factura Fiscal Mexico",
+					ffm_name,
+					"fm_pdf_custom_section",
+					text or "",
+					update_modified=False,
+				)
+		except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)